app/src/components/dataset/NewDatasetButton.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
+
+import {
+  Alert,
+  Button,
+  Card,
+  Icon,
+  Icons,
+  PopoverTrigger,
+  View,
+} from "@arizeai/components";
+
+import { useNotifySuccess } from "@phoenix/contexts";
+
+import { CreateDatasetForm } from "./CreateDatasetForm";
+
+export function NewDatasetButton({
+  onDatasetCreated,
+}: {
+  onDatasetCreated: (datasetId: string) => void;
+}) {
+  const [error, setError] = useState<string | null>(null);
+  const [isOpen, setIsOpen] = useState(false);
+  const notifySuccess = useNotifySuccess();
+  return (
+    <PopoverTrigger
```

Comment:
this is just pulled from the dataset picker that was coupled to this button on the add dataset to traces dialog